ERCS/erc-7943.md

```diff
@@ -150,34 +145,34 @@ It provides a way to freeze or unfreeze assets held by a specific user. This is
 *   **Compatibility:** Designed as an interface layer compatible with existing base standards like [ERC-20](./eip-20.md), [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md). Implementations extend from [ERC-7943](./eip-7943) alongside their base standard interface.
-*   **Essential enforcement rules:** Includes `forceTransfer` and `setFrozen` as standard functions, acknowledging its importance for regulatory enforcement in the RWA space, distinct from standard transfers. Mandates access control for this sensitive function.
+*   **Essential enforcement rules:** Includes `forcedTransfer` and `setFrozen` as standard functions, acknowledging its importance for regulatory enforcement in the RWA space, distinct from standard transfers. Mandates access control for this sensitive function.
 *   **[ERC-165](./eip-165.md):** Ensures implementing contracts can signal support for this interface.
 
-As an example, an AMM pool or a lending protocol can integrate with [ERC-7943](./eip-7943.md) based [ERC-20](./eip-20.md) tokens by calling `isUserAllowed` or `isTransferAllowed` to handle these assets in a compliant manner. Enforcement actions like `forceTransfer` and `setFrozen` can either be called by third party entities or be integrated by external protocols to allow for automated and programmable compliance. Users can then expand these tokens with additional features to fit the specific needs of individual asset types, either with on-chain identity systems, historical balances tracking for dividend distributions, semi-fungibility with token metadata, and other custom functionalities.
+As an example, an AMM pool or a lending protocol can integrate with [ERC-7943](./eip-7943.md) based [ERC-20](./eip-20.md) tokens by calling `isUserAllowed` or `canTransfer` to handle these assets in a compliant manner. Enforcement actions like `forcedTransfer` and `setFrozen` can either be called by third party entities or be integrated by external protocols to allow for automated and programmable compliance. Users can then expand these tokens with additional features to fit the specific needs of individual asset types, either with on-chain identity systems, historical balances tracking for dividend distributions, semi-fungibility with token metadata, and other custom functionalities.
 
 ### Notes on naming:
 
-- **`forceTransfer`**: This term was selected for its neutrality. While names like "confiscation," "revocation," or "recovery" describe specific motivations, `forceTransfer` purely denotes the direct action of transferring assets, irrespective of the underlying reason.
-- **`setFrozen` / `getFrozen`**: These names were chosen for managing transfer restrictions.
+- **`forcedTransfer`**: This term was selected for its neutrality. While names like "confiscation," "revocation," or "recovery" describe specific motivations, `forcedTransfer` purely denotes the direct action of transferring assets, irrespective of the underlying reason.
```

Comment:
I'd add a note on forcedTransfer/canTransfer to leave trace of the fact that these name where chosen because of consistency with all the other three mentioned ERCs. I believe that the "Notes on naming" section is important to establish a bit of clarity in the RWA space.